pkg/grizzly/workflow.go

```diff
@@ -18,6 +18,11 @@ import (
 
 var interactive = terminal.IsTerminal(int(os.Stdout.Fd()))
 
+func isMultiResource(handler Handler) bool {
```

Comment:
Why not assume it is always a multiResource?

Reply:
because then everything needs to implement Diff/Apply logic. I want to find ways to remove the multi-resource distinction, and make them all do that, but only when I've worked out how to avoid lots of unnecessary code duplication. I have ideas.